ui/src/linking/linking-utils.ts

```diff
@@ -124,7 +124,7 @@ export function asTrackLayoutSwitchJointConnection({
 export function suggestedSwitchJointsAsLayoutSwitchJointConnections(
     ss: SuggestedSwitch,
 ): LayoutSwitchJointConnection[] {
-    const tracks = Object.entries(ss.trackLinks).flatMap(([locationTrackId, links]) =>
+    const tracks = objectEntries(ss.trackLinks).flatMap(([locationTrackId, links]) =>
```

Comment:
Näitä `Object.entries`->`objectEntries`-muutoksia tulee useampi. TypeScript ei tyypitä `Object.entries`iä mitenkään tarkemmin itse, koska rakenteellisessa tyyppijärjestelmässä tyyppi ei ole lupaus, että oliolla on *vain* tyypissä mainitut jäsenet; objectEntries taas olettaa, että kyllä on.

Eli näiden kanssa on teoriassa ihan mahdollista mennä metsään, mutta kokonaisuutena toivottavasti kuitenkin on enemmän hyötyä kuin haittaa sen oletuksen tekemisestä, että sisään ei nyt tule ylimääräisiä kenttiä oliossa.